### PR TITLE
[Feat] Issue #738 - Add @storybook/addon-a11y for Accessibility Testing

### LINKED ISSUE #738: Need of a11y test for the components.
## Description

Storybook has an a11y addon that I think should be added as it eases the a11y tests for each component and would also disclose the a11y problems in a better manner.

## Implementation

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by mcastre**
## Fixes
Fixes [#738](https://github.com/creativecommons/vocabulary/issues/738) by @Dhruvi16 

## Description
Adds @storybook/addon-a11y Storybook plugin in packages/vocabulary

## Technical details
Currently, the a11y addon is set to all stories in the Vocabulary package. To disable the addon for a specific story, you can add to the `parameters` in the `<Meta>` tag to disable it. 
Example:
`<Meta title="Navigation/Breadcrumb" parameters={{ a11y: { disable: true } }/>`

## Tests
All unit tests passed.

## Screenshots
Example of a passing Accessibility test on a Story:
image

Example of a failing Accessibility test:
image

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
   "author": "Creative Commons (https://creativecommons.org)",
   "workspaces": {
     "packages": [
-        "packages/*"
+      "packages/*"
     ]
-},
+  },
```
Comment (dhruvkb) [inline package.json]: These aren't valid changes as we follow a two-space indentation rule in the project.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
1. Don't install any packages in root `package.json`.
2. Please also install the addon in the Vue Vocabulary storybook.

File: package.json (hunk 1 above)
Comment (mcastre) [reply package.json]: @Dhruvi16 sorry that was not on purpose. Will revert

**Comment by mcastre** [comment]
@Dhruvi16 @dhruvkb PR has been updated with requested changes.

File: package.json [hunk 2]
```diff
@@ -3,9 +3,9 @@
   "private": true,
   "author": "Creative Commons (https://creativecommons.org)",
   "workspaces": {
-    "packages": [
-        "packages/*"
-    ]
+  "packages": [
```
Comment (Dhruvi16) [inline package.json:6]: These changes should not reflect in your PR.

Comment (mcastre) [reply package.json:6]: I was asked to revert my changes to this file, even though it resolved the spacing issues

**Review by zackkrida — APPROVED** [review]

**Comment by zackkrida** [comment]
Thank you @mcastre, I'm going to keep the spacing fixes you made to the root `package.json`.

**Review by zackkrida — APPROVED** [review]

**Pull request merged**